Scripts/Multiplayer/NET_manager.gd

```diff
@@ -11,6 +11,11 @@ signal chat_message_arrived(content : String)
 signal chat_log_cleared
 
 
+func _process(_delta) -> void:
+	if Input.is_action_just_pressed("FORCE_DESYNC"):
```

Comment:
I'd keep input in input manager
having `Input.is_action_just_pressed` scattered around code base is messy

Reply:
xd, it is temporary debug which will not go to master

Reply:
i'd keep this one it can be useful for testing, maybe as a cheat in the chat